matlab/JSBSimInterface.cpp

```diff
@@ -0,0 +1,954 @@
+#include "JSBSimInterface.h"
+#include <models/FGAircraft.h>
+#include <models/FGAccelerations.h>
+#include <math/FGQuaternion.h>
+
+/* 2021-03-21 compiles with JSBSim 1.1.5
+ * Compiles with JSBSim checkout a59596f8f0d4c7b4f7ba24e99997bc2071d6cb72
+ */
+
+JSBSimInterface::JSBSimInterface(FGFDMExec *fdmex)
+{
+	_ac_model_loaded = false;
+	fdmExec = fdmex;
```

Comment:
```suggestion
	fdmExec = new FGFDMExec;
```
Create an instance of `FGFDMExec` that will de destroyed as soon as `JSBSimInterface` is deleted.